### ISSUE TITLE
Troubleshooting: Pantheon Solr module

### ISSUE THREAD

**Issue by eabquina** [pantheon-systems/documentation#3764]
Re: [Apache Solr on Pantheon](https://pantheon.io/docs/solr-drupal-7/)

Priority: Low

## Issue Description:
Case ID: 107903

From a customer: 
Pantheon Solr module prevents using a non-Pantheon server, but it disguises the fact very well and allows you to easily delete your entire index.

You can create a new search environment, select content types to index and set it running and will run. But no data is ever sent to the non-Pantheon/external Solr server,  It is being sent in the internal search index. 

When de-selecting or resting the external Solr index, it deletes the data from your local index, losing data.

In summary; Pantheon Solr has to be disabled in order to use an external Solr server, they can't run in parallel. 

## Suggested Resolution

Include as a watch out or troubleshooting section about using External Index servers

**Comment by alexfornuto** [comment]
For clarity, this is referencing the Drupal module.

**Comment by alexfornuto** [comment]
Reaching out to @stevector for technical validation.

**Comment by stevector** [comment]
I wrote the Drupal 8 module but have less experience with the Drupal 7 module. Validating/resolving this would require setting up an external Solr service and trying to reproduce. cc @greg-1-anderson

**Comment by greg-1-anderson** [comment]
It is true that our Solr modules are not designed to work on non-Pantheon Solr servers. I don't think it is necessary to reproduce the behavior in the report, or describe the symptoms in detail in the documentation. It would be sufficient to warn the users that using a non-Pantheon server will produce undefined results, and warn that the Pantheon Solr module should be uninstalled before using an external Solr server.
